nautobot_golden_config/nornir_plays/processor.py

```diff
@@ -56,9 +50,3 @@ def task_instance_completed(self, task: Task, host: Host, result: MultiResult) -
             exception_string = ", ".join([str(e[0]) for e in exceptions])
             # Log only exception summary to users
             self.logger.error(f"{task.name} failed: {exception_string}", extra={"object": task.host.data["obj"]})
-            for exception in exceptions:
```

Comment:
Can you explain this one, not sure I understand it yet?

Reply:
I believe this was there before but this particular piece is reviewing all the tasks for a Host and will log if a failure is found. This is easier than trying to review the full AggregatedResult.